tests/test___main__.py

```diff
@@ -25,4 +26,28 @@ def test_consume():
     consume(mock_change)
     assert mock_writer.put_message.called, 'Sanity'
     assert call.cursor.send_feedback(flush_lsn=10) in mock_change.mock_calls, \
-        'we sent feedback!'
\ No newline at end of file
+        'we sent feedback!'
+
+
+    mock_time = Mock()
```

Comment:
Please move this into a new function.

Reply:
Happy to do that but there is a fair amount of shared setup here and the convention in the tests is one test per method. It seems like either the underlying method should be factored and a new test created, or this should be left as is.

Thoughts?

Reply:
Fair.